erpnext_com/www/docs/user/manual/en/support/issue.md

```diff
@@ -16,6 +16,18 @@ Frappe system.
 
 <img class="screenshot" alt="Issue" src="{{docs_base_url}}/assets/img/support/issue.png">
 
+#### Service Level Agreement
+
+If Service Level Agreements are enabled, you can track it in Issues. Every Issue
```

Comment:
The GIFs have inconsistent padding, crop them to be the same as screenshots i.e., just a little on either side
Use realistic data for the gifs and screenshots, use realistic names, issues, data etc